pysatNASA/instruments/de2_wats.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
-"""Supports the Wind and Temperature Spectrometer (WATS) instrument on
+"""Module for the DE2 WATS instrument.
+
```

Comment:
This comment is just for fun. I searched for the WATS reference since last I checked it doesn't exist anymore. Searching for `J. C. Maurer, The Dynamics Explorer Wind and Temperature Spectrometer
Space Sci. Instrum., v. 5, n. 4, p. 417, 1981.` brought up pysat 2.3.0 as the first result. :)

Reply:
I found it in ADS, but with a different lead author: https://ui.adsabs.harvard.edu/abs/1981SSI.....5..417S/abstract

Reply:
updating format for consistency elsewhere

Reply:
That's what I get for incomplete jokes in review comments, unintentionally create work. I wasn't complaining about your reference but rather that Space Sci. Instrum. disappeared and took the articles with it. That was at least my last understanding. Feel free to ignore.

Reply:
Nope, it's fixed.  I noticed that 3 out of 4 DE2 instruments used a weird formatting (vol, issue, first page) rather than the standard (vol, page range) everything used.

Reply:
No worries, initially didn't see the joke until I started poking into it